recipes/openmm/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "openmm" %}
+{% set version = "7.4.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: cb7ddd923d6f402cf2723b3c14509915abad328505e8ce401ac3131706dabeff
+
+build:
+  number: {{ build }}
+  skip: True  # [py2k or win]
+  missing_dso_whitelist:
+  - "*/libcuda.*"  # [linux]
```

Comment:
Why is this needed?

Reply:
Because we've enabled `error_overlinking` on staged-recipes and since `/usr/local/lib/libcuda.so` is outside of prefix, conda-build errors. Correct way to fix this would be to generate the stub library `libcuda.so` ourselves and make that a CDT.

Reply:
Can you please raise an issue about this somewhere so we can follow-up? Maybe the `nvcc` feedstock or the webpage repo?

Reply:
This is now being tracked in issue ( https://github.com/conda-forge/nvcc-feedstock/issues/12 ).